releasenotes/notes/cni-host-network.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: feature
+area: traffic-management
+issue:
+  - 33944
+
+releaseNotes:
+  - |
+    **Removed** `hostNetwork` enablement from `istio-cni-node` DaemonSet.
```

Comment:
Do we really need the release note at all? It seems like an implementation detail, should it affect users? I don't think so.

Reply:
I feel this might be useful if user have security policy set up to give hostNetwork privilege to cni DaemonSet. Now they can trim that. But anyway I don't feel strong about adding this. Removed.